src/main/java/com/minecolonies/coremod/colony/buildings/AbstractSchematicProvider.java

```diff
@@ -369,7 +369,11 @@ public int getRotation()
 
         try
         {
-            final Blueprint blueprint = StructurePacks.getBlueprint(this.structurePack, this.path, true);
+            Blueprint blueprint = StructurePacks.getBlueprint(this.structurePack, this.path, true);
+            if (blueprint == null && this.path.endsWith("0.blueprint"))
```

Comment:
It looks like paths of pasted hut blocks don't end in ".blueprint" at the moment. The log of the discord report also mentions that the path is "military/altbarrackstower0". So this `if` condition may still fail in that case.
(Not sure if that is a bug or an intended change on Structurize's side)